controllers/past-grants/views/index.njk

```diff
@@ -37,15 +49,22 @@
                                name="q"
                                value="{{ queryParams.q }}"
                                placeholder="Enter a search term, location, or postcode"
-                               v-on:change="filterResults"
+                               autocomplete="nope"
```

Comment:
This is weird but [recommended by MDN](https://developer.mozilla.org/en-US/docs/Web/Security/Securing_your_site/Turning_off_form_autocompletion).

Reply:
Love it